### PR TITLE
feat added authentication

### PR DESCRIPTION
**Pull request by varnita619**
added login, logout, signup authentication

Screenshot (33)

### CODE REVIEW

File: src/pages/Auth/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+              }
+              autoComplete="true"
+              required
+            />
+
+            <label>
+              <input
+                type="checkbox"
+                name="remember"
+                style={{ marginBottom: "15px" }}
```
Comment (rohit8686) [inline src/pages/Auth/Login.jsx:90]: better to avoid inline styling

**Review by rohit8686 — APPROVED** [review]
remove AuthReducer.js if it will not be used and rest of code looks clean, good to merge

File: src/Components/VideoCard/VideoCard.css [hunk 2]
```diff
@@ -4,7 +4,7 @@
     align-items: flex-start;
     justify-content: flex-start;
     text-align: start;
-    width: max-content;
+    /* width: max-content; */
```
Comment (anuj-kumary) [inline src/Components/VideoCard/VideoCard.css:7]: Remove unwanted comment

File: src/pages/Auth/Auth.css [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  opacity: 1;
+}
+
+.login-button {
+  background-color: var(--secondary-color);
+  margin: 1rem 0;
+  border: none;
+  cursor: pointer;
+  opacity: 0.9;
+  padding: 1rem 1rem 1rem 1rem;
```
Comment (anuj-kumary) [inline src/pages/Auth/Auth.css:20]: You can give `Padding: 1rem` here it work

**Review by anuj-kumary — APPROVED** [review]

**Pull request merged**